jvm-packages/xgboost4j-gpu/src

```diff
@@ -1 +0,0 @@
-../xgboost4j/src/
```

Comment:
I remove the `src/` symlink, so that the `xgboost4j-gpu` package can have an indepedent `resources/` directory that's distinct from the `resources/` directory of the `xgboost4j` package.